trailblazer/store/models.py

```diff
@@ -23,6 +21,14 @@ class Info(Model):
     created_at = Column(types.DateTime, default=datetime.datetime.now)
     updated_at = Column(types.DateTime)
 
+    def to_dict(self) -> dict:
```

Comment:
`Alchy` added these `to_dict` methods by default, so they need to be implemented manually now. We do it in a hacky way in cg, but I prefer this explicit way. There are many other options for serialization in Python, for instance the `Marsmallow` package.

I feel like we should standardize which tools we use for these things and document it. Since we are quite few developers compared to the amount of code we maintain, it is good if we settle on using one solution for solving common problems like
- What do we use for serialization?
- What do we use for CLI:s?
- What ORM do we use?
etc.

Reply:
Also, let me know if some of the fields should be excluded. Have not looked through them all yet